app/src/main/resources/common/README.md.mustache

```diff
@@ -91,14 +91,28 @@ On a successful deployment via the following methods, the server will be availab
 {{#configServer}}* `https:/localhost:8888/casconfigserver`{{/configServer}}
 {{#discoveryServer}}* `https://localhost:8761`{{/discoveryServer}}
   
-## Executable WAR
+## Build and run CAS with an embedded server
 
-Run the server web application as an executable WAR. Note that running an executable WAR requires CAS to use an embedded container such as Apache Tomcat, Jetty, etc.
+Run the server web application as an executable WAR. 
+
+Note that running an executable WAR requires CAS to use an embedded container such as Apache Tomcat, Jetty, etc.
 
 {{#appServer}}The current servlet container is specified as `{{appServer}}`.{{/appServer}}
 {{^appServer}}No servlet container is specified for the current build. Examine your `gradle.properties` file
 and modify the `appServer` property to point to the appropriate container of choice.{{/appServer}}
 
+### Building
+
+In order to build an executable war, update `gradle.properties` and set `executable=false`, then run:
```

Comment:
This is not maintainable; manual instructions like this in the README are not going to be looked after by maintainers. Explanations like this should go into the gradle.properties file where the change actually belongs.